maven-surefire-report-plugin/src/main/java/org/apache/maven/plugins/surefire/report/AbstractSurefireReportMojo.java

```diff
@@ -34,13 +27,21 @@
 import org.apache.maven.shared.utils.PathTool;
 import org.apache.maven.shared.utils.StringUtils;
 
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Locale;
+import java.util.ResourceBundle;
+
 /**
  * Abstract base class for reporting test results using Surefire.
  *
  * @author Stephen Connolly
  */
 public abstract class AbstractSurefireReportMojo
-    extends AbstractMavenReport
+        extends AbstractMavenReport
```

Comment:
Do not change lines only because of checkstyle. Do it again and fix only lines for the purpose of this issue only.